README.md

```diff
@@ -23,6 +23,16 @@ rocks trees.
 * [Installation instructions for macOS](http://luarocks.org/en/Installation_instructions_for_macOS)
 * [Installation instructions for Windows](http://luarocks.org/en/Installation_instructions_for_Windows)
 
+## Contributing
+
+* Create a new branch with a meaningful name `git checkout -b branch_name`.<br />
+* Develop your feature on Xcode IDE  and run it .<br />
+* Add the files you changed `git add file_name`.<br />
+* Commit your changes `git commit -m "Message briefly explaining the feature"`.<br />
```

Comment:
instead of using `-m`, it's better to let git open the text editor so that a longer explanation can be entered in the commit.

Reply:
I will make the changes!